core/go/build.gradle

```diff
@@ -31,7 +31,7 @@ ext {
         include "mocks/**/*.go"
     }
 
-    targetCoverage = 94.0
+    targetCoverage = 93.5
```

Comment:
[Update coverage (no drop in PR)](https://github.com/LF-Decentralized-Trust-labs/paladin/pull/573/commits/55989ec0089e062db447fc078292d261548ccd4c)

it's not clear why this coverage drop is needed if there is no coverage drop caused by this PR